src/foreman_mcp_server/server.py

```diff
@@ -39,51 +54,68 @@ def normalize_log_level(_ctx, _param, value):
 )
 @click.option(
     "--foreman-username",
-    default="admin",
-    help="Username for Foreman API authentication",
+    help="Username for Foreman API authentication. Can be set via FOREMAN_USERNAME environment variable.",
     envvar="FOREMAN_USERNAME",
 )
 @click.option(
     "--foreman-password",
-    default="changeme",
-    help="Password for Foreman API authentication",
+    help="Password for Foreman API authentication. Can be set via FOREMAN_PASSWORD environment variable. Personal access tokens are recommended.",
     envvar="FOREMAN_PASSWORD",
 )
 @click.option(
     "--transport",
     default="streamable-http",
     help="Transport protocol to use (streamable-http, stdio)",
 )
+@click.option(
+    "--verify-ssl",
+    default=True,
+    is_flag=True,
+    help="Verify SSL certificates when connecting to Foreman API.",
+)
```

Comment:
This is weird, it is enabled by default and passing in `--verify-ssl` flips it to false.

```suggestion
@click.option(
    "--verify-ssl/--no-verify-ssl",
    default=True,
    is_flag=True,
    help="Verify SSL certificates when connecting to Foreman API.",
)
```